apps/sms/js/thread_list_ui.js

```diff
@@ -364,8 +449,17 @@ var ThreadListUI = {
         MessageManager.markThreadRead(thread.id, isRead);
       }
     });
-
+    
     this.cancelEdit();
+
+    isRead ? l10nKey1 = 'marked-as-read' : l10nKey1 = 'marked-as-unread';
```

Comment:
nit: better write it: `l10nKey = isRead ? 'marked-as-read' : 'marked-as-unread';`